tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -773,7 +773,11 @@ def pack_params_to_args_namespace(**kwargs):
     for arg in vars(argv):
```

Comment:
Can you please add description for this function? Not cleat what it does?

Reply:
Added comments to pack_params_to_args_namespace() method.